### ISSUE TITLE
How to use this code when infer?

### ISSUE THREAD

**Issue by Bin-ze** [chou141253/FGVC-PIM#13]
I checked the code carefully. You write the loss calculation in the model. After initializing the model, the model needs to be sent to the label. There is no code design for infer, and after the end-to-end training, a series of post-processing is used. , including: splicing features, fusion features, etc., and then classify them separately, I want to know if I don't know gt, how to choose the best result? Is this code just written to brush the list? Where is the logic of the actual application?

**Comment by chou141253** [comment]
"We adopt a very simple design, which passes each feature point through a fully connected layer to predict the category.
When the highest probability of the predicted result after softmax is greater than a certain value, the feature point
is considered as a helpful feature and will be reserved for subsequent fusion. Conversely, the feature point is considered less helpful for fine-grained classification." -- [https://arxiv.org/pdf/2202.03822v1.pdf](https://arxiv.org/pdf/2202.03822v1.pdf)

To know which area is discriminative or not, the PIM module does not need the ground truth.

By the way, we update our code. Hope the new version can provide more clearer explanation. Thanks.
